src/NetPanzer/Views/Game/VehicleSelectionView.cpp

```diff
@@ -590,11 +590,11 @@ void VehicleSelectionView::drawMiniProductionStatus(Surface &dest)
             {
                 strncpy(strBuf, obj->occupying_player->getName().c_str() , 17);
```

Comment:
so basically this is a mechanism to trim the player's name. If we switch this over to cpp strings we can just use the built in trim() method instead of maintaining more buffers etc, I suspect it would get optimized better, too. Probably out of scope for this PR though.

Reply:
I tried using what I found for the built-in trim method but was told it's out of scope. I temporarily created a truncate function.

Reply:
nvm I got rid of the truncate function